.github/workflows/auto-merge.yml

```
@@ -0,0 +1,20 @@
+name: Auto-merge Dependabot PRs
+on:
+  pull_request:
+    types: [opened, ready_for_review]
+    branches: [main]
+jobs:
+  auto-merge:
+    if: github.actor == 'dependabot[bot]'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Enable auto-merge
+        uses: actions/github-script@v6
```

actionlint (1.7.7)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)